StoreTFstateAzure.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+RESOURCE_GROUP_NAME="CreditSuisse1_FengPan_ProjectExercise"
+STORAGE_ACCOUNT_NAME="tstatefeng"
+CONTAINER_NAME="tstatecon"
+
+# Create storage account
+az storage account create --resource-group $RESOURCE_GROUP_NAME --name $STORAGE_ACCOUNT_NAME --sku Standard_LRS --encryption-services blob
+
+# Get storage account key
+ARM_ACCESS_KEY=$(az storage account keys list --resource-group $RESOURCE_GROUP_NAME --account-name $STORAGE_ACCOUNT_NAME --query '[0].value' -o tsv)
+
+# Create blob container
+az storage container create --name $CONTAINER_NAME --account-name $STORAGE_ACCOUNT_NAME --account-key $ARM_ACCESS_KEY
+
+echo "storage_account_name: $STORAGE_ACCOUNT_NAME"
+echo "container_name: $CONTAINER_NAME"
+echo "access_key: $ACCOUNT_KEY"
```

Comment:
And for an uncommented script, printing out an access key feels like scary behaviour!